dataduct/data_access/connection.py

```diff
@@ -7,50 +7,73 @@
 
 from ..config import Config
 from ..utils.helpers import retry
+from ..utils.helpers import exactly_one
 from ..utils.exceptions import ETLConfigError
 
 config = Config()
+CONNECTION_RETRIES = config.etl.get('CONNECTION_RETRIES', 2)
 
-@retry(2, 60)
-def redshift_connection(**kwargs):
-    """Fetch a psql connection object to redshift
+
+def get_redshift_config():
+    """Get redshift config from config file and return the dictionary
```

Comment:
Yeah I just didn't have enough time, I'll fix them in some other diff going forward.